### ISSUE TITLE
Ad Astra's Making Iron Sheets quest bugged

### ISSUE THREAD

**Issue by DarrylSSY** [LunaPixelStudios/Prominence#336]
### Describe the Bug

The quest requires 6x Iron Plate from Tech Reborn, which has been removed from Prominence 2 and replaced with Iron Sheets from Create.

There is currently no way to get the Iron Plate from Tech Reborn. Tried the following but it all ended up with Create's Iron Sheets:
- Iron Ingot + Hammer
- Compressing Iron Ingot through all the different kinds of compressors

Having 6x of Create's Iron Sheet does not clear the quest either.

### Reproduction Steps

1. Craft 6x Iron Sheets (Hammer + Iron Ingot)
2. Open quest book > Ad Astra > Making Iron Sheets
3. Quest is shown as not completed

### Screenshots and Videos

![image](https://github.com/LunaPixelStudios/Prominence/assets/5838225/61f36be0-7973-464d-9ea3-0a000ced5a81)

### Operating System

Windows 11

### Minecraft Version

Prominence II 1.20.1

### Modloader

Fabric

### Modpack Version

2.0.0

### Optifine

No

**Comment by deniial00** [comment]
Yeah, I had the same problem on a private server - only way to go around that problem is to go into creative and give yourself 6 plates.
